packages/components/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -228,3 +236,38 @@ export const Fitted = (): JSX.Element => {
     </Tabs>
   );
 };
+
+export const WithDivider = (): JSX.Element => {
```

Comment:
Typically when creating stories or really doing anything with storybook/component libraries, I like to think about how can I make it easy for someone looking at this to be able to quickly understand and get going with this.

In this case, we could make it easier to understand how exactly to use it with the divider by adding a description to the story that says something like "You can display a divider between the tabs and the content by using the `withDivider` property on the `TabList` component."

(I noticed the docs page isn't working for this component so this won't actually be visible until it's fixed but still good to have for when we do fix it)